challenges/2021-04-20-potential-palindrome/challenge.md

```diff
@@ -0,0 +1,32 @@
+# Potential Palindrome
+
+Given a string, determine whether any permutation of it is a palindrome. A palindrome is any string that can be read the same both forwards and backwards, such as "kayak".
+
+## Business Rules/Errata
+
+- You only need to determine whether it is possible to make a palindrome, you do not need to return an example of the palindrome.
+- Any input that is not a string should be treated as a string, if possible.
+- An empty string is not a palindrome.
+
+## Examples
+
+```bash
+canMakePalindrome("carrace");  // True
+```
+
+The string `carrace` can be rearranged to make `racecar`, a palindrome.
+
+## Tackling This Challenge
+
+1. Make sure you've got the required software on your machine: A JDK 11+
+1. If you haven't already, fork the CodingDojo repository ([INSTRUCTIONS](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo)).
+1. Checkout a new branch using `git checkout -b yourgithubusername-wip`.
+1. Add your code to the 'mob.java' file to make the `countWordsBetween` function work as expected.
```

Comment:
I am guessing this needes to be canMakePalindrome instead of countWordsBetween

Reply:
Yep.